spec/duration.html

```diff
@@ -1289,27 +1289,33 @@ <h1>RoundDuration ( _years_, _months_, _weeks_, _days_, _hours_, _minutes_, _sec
           1. Let _monthsWeeksInDays_ be ? DaysUntil(_yearsLater_, _yearsMonthsWeeksLater_).
           1. Set _relativeTo_ to _yearsLater_.
           1. Let _days_ be _days_ + _monthsWeeksInDays_..
+          1. Let _daysDuration_ be ? CreateTemporalDuration(0, 0, 0, _days_, 0, 0, 0, 0, 0, 0).
+          1. Let _daysLater_ be ? Call(_dateAdd_, _calendar_, « _relativeTo_, _daysDuration_, _options_, %Temporal.PlainDate% »).
```

Comment:
`_options_` is currently undefined if `_relativeTo_` is undefined, but an empty object otherwise; mind fixing that while you're here for correctness/consistency throughout RoundDuration?

Reply:
This is a bug in the spec. The polyfill throws if `unit` is `"years"` and there is no `calendar` (basically no `relativeTo`). Should I fix it here or in a second PR?

Reply:
Second PR is fine, thanks!